Add white belt (#1)
AppSecure-nrw/security-belts: white/continuous-security-improvement.md

> # Continuously Improve your Security Belt

Author: ```suggestion
# Continuously Improve your Security Belt Rank
```

> Security is like a big elephant. You cannot eat it as a whole, but you can slice it and continuously eat small pieces. To make this happen, ensure that the Product Owner continuously prioritizes security belt activities within the product backlog. Security belt activities make good pieces because they are of reasonable size and have a defined output. Celebrate all your implemented security belt activities.

Author: ```suggestion
Security is like a big elephant. You cannot eat it as a whole, but you can slice it and continuously eat small pieces. To make this happen, ensure that the Product Owner continuously prioritizes security belt activities for the next belt within the product backlog. Security belt activities make good pieces because they are of reasonable size and have a defined output. Celebrate all your implemented security belt activities.
```

> Security is like a big elephant. You can not eat it as a whole, but you can slice it and continuously eat small peaces. To make this happen, ensure that the Product Owner continuously prioritizes effort of the team to work on topics for the next security belt. Of cause Product Owners shall only prioritize small chunks of work within the product backlog to be able to react to changed requirements. Security belt activities make good peaces to be put into the product backlog, because they are of reasonable size and have a defined output.

Author: ```suggestion
Security is like a big elephant. You cannot eat it as a whole, but you can slice it and continuously eat small pieces. To make this happen, ensure that the Product Owner continuously prioritizes effort of the team to work on topics for the next security belt. Of course, Product Owners shall only prioritize small chunks of work within the product backlog to be able to react to changed requirements. Security belt activities make good pieces to be put into the product backlog because they are of reasonable size and have a defined output.
```

# Continuously Improve your Security Belt

> Security is like a big elephant. You cannot eat it as a whole, but you can slice it and continuously eat small pieces. To make this happen, ensure that the Product Owner continuously prioritizes effort of the team to work on topics for the next security belt.
> 
> Of course, Product Owners shall only prioritize small chunks of work within the product backlog to be able to react to changed requirements. Security belt activities make good pieces to be put into the product backlog because they are of reasonable size and have a defined output.

Author: ```suggestion
Security is like a big elephant. You cannot eat it as a whole, but you can slice it and continuously eat small pieces. To make this happen, ensure that the Product Owner continuously prioritizes security belt activities within the product backlog. Security belt activities make good pieces because they are of reasonable size and have a defined output. Celebrate all your implemented security belt activities.
```
